landing-zone/example.tfvars

```diff
@@ -0,0 +1,484 @@
+##############################################################################
+# Account Variables
+# > These will apply to any resources created using this template
+##############################################################################
+
+ibmcloud_api_key = "<USER DEFINED INPUT>"
```

Comment:
For discussion, I think this one is fine to add to the detect secrets ignore list

Reply:
Added to secrets base line